src/libraries/System.Drawing.Common/ref/System.Drawing.Common.cs

```diff
@@ -305,7 +305,7 @@ public sealed partial class Font : System.MarshalByRefObject, System.ICloneable,
         [System.ComponentModel.DesignerSerializationVisibilityAttribute(System.ComponentModel.DesignerSerializationVisibility.Hidden)]
         public string Name { get { throw null; } }
         [System.ComponentModel.BrowsableAttribute(false)]
-        public string OriginalFontName { get { throw null; } }
+        public string? OriginalFontName { get { throw null; } }
```

Comment:
FYI @safern @tannergooding

Reply:
Will defer to @safern here as I'm still familiarizing myself with the code.

I did notice that all the public entry points I looked at which assign `OriginalFontName` look to throw (generally via the P/Invoke) if the member is null, so I'm not sure marking it nullable is correct.

Reply:
Ah, but I missed that we catch that exception and reassign the `family` without resetting `OriginalFontName` 👍

Reply:
I'm confused about this... the implementation for this is:

 public string OriginalFontName => _originalFontName;

and as @tannergooding `_originalFontName` would never be null as we don't allow to initialize with a null font family and if we get an exception from the P/Invoke we fallback to a default `_fontFamily`.

When will `_originalFontName` ever be null?

Reply:
We will assign it here, with no check: https://source.dot.net/#System.Drawing.Common/System/Drawing/Font.Unix.cs,50

https://source.dot.net/#System.Drawing.Common/System/Drawing/Font.Unix.cs,56 will then throw
and we will catch and reassign `family`, but not change `_originalFontName`

Reply:
Ah that's right. It seems like this is not called from any public code path (Only the serialization constructor and an internal code path that never passes down null) on Unix -- On windows it seems like we indeed validate for non-null.

Reply:
Thanks. So in summary, the change makes sense?

Reply:
Yeah, makes sense.